packages/account/src/Components/account-limits/withdrawal-limits-table.tsx

```diff
@@ -0,0 +1,142 @@
+import { ButtonLink, Text } from '@deriv/components';
+import { formatMoney, isMobile } from '@deriv/shared';
+import { Localize, localize } from '@deriv/translations';
+import AccountLimitsTableCell from './account-limits-table-cell';
+import AccountLimitsTableHeader from './account-limits-table-header';
+import React from 'react';
+import { observer, useStore } from '@deriv/stores';
+
+const AccountLimitsWithdrawalContent = ({ is_appstore }: { is_appstore: boolean }) => {
+    return is_appstore ? (
```

Comment:
do we need 'is_appstore'? :)

Reply:
@yauheni-deriv, I heard this was going to be removed, but I can't take it out right now, because no one actually told me to do so